src/sagemaker_training/modules.py

```diff
@@ -129,6 +129,23 @@ def install(path, capture_error=False):  # type: (str, bool) -> None
     )
 
 
+def install_requirements(path, capture_error=False):  # type: (str, bool) -> None
```

Comment:
don't we need unit test for that?

Reply:
Added a unit tests. The integration tests also covers this functionality.